incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -220,7 +222,7 @@ func (r *HierarchyConfigReconciler) updateFinalizers(ctx context.Context, log lo
 // be able to proceed until this one is finished. While the results of the reconiliation may not be
 // fully written back to the apiserver yet, each namespace is reconciled in isolation (apart from
 // the in-memory forest) so this is fine.
```

Comment:
Document return values? Do we really need two or do they just get "or'd" together?